app/models/user.rb

```diff
@@ -48,7 +82,12 @@ def format_phone_number
   end
 
   def send_phone_confirmation
-    ApplicationController.helpers.send_confirmation_text(self)
+    if use_text? && self.text_confirmation_sent_at.blank?
+      ApplicationController.helpers.send_confirmation_phone(self, true)
```

Comment:
Why is the ApplicationController responsible for sending the phone confirmation? Would the `send_confirmation_phone` method be better in a Twilio helper or something?

Reply:
In this case, we could just import the respective helper into this model. That's what they're there for!